src/interfaces/curses/ec_curses_mitm.c

```diff
@@ -60,13 +60,17 @@ struct wdg_menu menu_mitm[] = { {"Mitm",                'M', "", NULL},
 static void curses_arp_poisoning(void)
 {
    char *method = "arp:";
-   size_t len = strlen(method);
+   size_t len1 = strlen(method);
+
+   char *default_param = "remote";
+   size_t len2 = strlen(default_param);
```

Comment:
`method` and `default_param` are both initialized on the stack then copied into another stack buffer concatenated.
I don't see the benefit having the param and method in two separate variables and copy twice.
Either use `snprintf()` to perform the copy in one shot or I'd prefer, having one stack variable per MITM method (ARP|NDP) and copy it once into the global `params` buffer.